src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -23,6 +23,7 @@ interface PharmacyProps {
 }
 
 export function PharmacyCard(props: PharmacyProps) {
+  /*
```

Comment:
Might be worthwhile to just delete dead code, it just adds noise to a codebase.

Besides the `PharmacyList` container component should responsible for maintaining booking state, so having this component also manage booking state confuses as to where the source of truth is.